Doesn't work when imported via HTTP or HTTPS URL

Currently deno_exif doesn't seem to work when trying to import it from an HTTP(S) URL like so:

```ts
import { create } from 'https://deno.land/x/deno_exif@0.0.2/mod.ts';
```

If trying to use it with the above import I always get the following error:

![image](https://user-images.githubusercontent.com/75155/178122917-3962ccd5-83e9-477e-8c4a-d2d2ac4443e2.png)

<details>
<summary>Expand this to see the error as plain text</summary>

```
error: Uncaught (in promise) InvalidData: invalid url scheme
    throw new Deno.errors.InvalidData("invalid url scheme");
          ^
    at fileURLToPath (https://deno.land/std@0.88.0/node/url.ts:53:11)
    at createRequire (https://deno.land/std@0.88.0/node/module.ts:533:18)
    at https://deno.land/x/deno_exif@0.0.2/mod.ts:4:17
```

</details>

However if I clone the repository for example under a `lib/` dir in my project I can import it like this and it works without errors:

```ts
import { create } from './lib/deno_exif/mod.ts';
```

Using a `file://` URL also seems to work fine:

```ts
import { create } from 'file:///home/haprog/repos/cli-scripts/lib/deno_exif/mod.ts';
```

I also tested running a local HTTP server and importing from there with:

```ts
import { create } from 'http://127.0.0.1:8081/mod.ts';
```

But that also resulted in the same error above.

So for some reason this only seems to work when importing from local file system path but not when importing via HTTP or HTTPS.

Tested with deno 1.23.3.

Thanks for quick response but unfortunately it still doesn't seem to work with 0.0.3 though now the error is different.

If I use 0.0.3 with:

```ts
import { create } from 'https://deno.land/x/deno_exif@0.0.3/mod.ts';
```

I get:

```
error: Uncaught (in promise) Error: Cannot find module './lib/parser.js'
Require stack:
- /home/haprog/repos/cli-scripts/testExifSimple.ts
      const err = new Error(message) as Error & {
                  ^
    at Function._resolveFilename (https://deno.land/std@0.88.0/node/module.ts:270:19)
    at Function._load (https://deno.land/std@0.88.0/node/module.ts:377:29)
    at Module.require (https://deno.land/std@0.88.0/node/module.ts:130:21)
    at require (https://deno.land/std@0.88.0/node/module.ts:1141:16)
    at https://deno.land/x/deno_exif@0.0.3/mod.ts:5:16
```

I think it is now trying to resolve the required files relative to my project dir instead of relative to the import URL. If I run my script within my local clone of deno_exif then it works even when importing via the above URL because I assume it just loads the rest of the files from local paths.

--- a/lib/bufferstream.js
+++ b/lib/bufferstream.js
@@ -85,4 +85,4 @@ BufferStream.prototype = {
 	}
 };
 
-module.exports = BufferStream;
+export default BufferStream;

--- a/lib/date.js
+++ b/lib/date.js
@@ -77,8 +77,8 @@ function parseExifDate(dateTimeStr) {
 	}
 }
 
-module.exports = {
-	parseDateWithSpecFormat: parseDateWithSpecFormat,
-	parseDateWithTimezoneFormat: parseDateWithTimezoneFormat,
-	parseExifDate: parseExifDate
+export default {
+	parseDateWithSpecFormat,
+	parseDateWithTimezoneFormat,
+	parseExifDate,
 };

--- a/lib/exif-tags.js
+++ b/lib/exif-tags.js
@@ -1,4 +1,4 @@
-module.exports = {
+export default {
 	exif : {
 		0x0001 : "InteropIndex",
 		0x0002 : "InteropVersion",

--- a/lib/exif.js
+++ b/lib/exif.js
@@ -115,7 +115,7 @@ function readHeader(stream) {
 	return tiffMarker;
 }
 
-module.exports = {
+export default {
 	IFD0: 1,
 	IFD1: 2,
 	GPSIFD: 3,

--- a/lib/jpeg.js
+++ b/lib/jpeg.js
@@ -1,6 +1,6 @@
 /*jslint browser: true, devel: true, bitwise: false, debug: true, eqeq: false, es5: true, evil: false, forin: false, newcap: false, nomen: true, plusplus: true, regexp: false, unparam: false, sloppy: true, stupid: false, sub: false, todo: true, vars: true, white: true */
 
-module.exports = {
+export default {
 	parseSections: function(stream, iterator) {
 		var len, markerType;
 		stream.setBigEndian(true);

--- a/lib/parser.js
+++ b/lib/parser.js
@@ -1,8 +1,9 @@
 /*jslint browser: true, devel: true, bitwise: false, debug: true, eqeq: false, es5: true, evil: false, forin: false, newcap: false, nomen: true, plusplus: true, regexp: false, unparam: false, sloppy: true, stupid: false, sub: false, todo: true, vars: true, white: true */
 
-var jpeg = require('./jpeg'),
-	exif = require('./exif'),
-	simplify = require('./simplify');
+import jpeg from './jpeg.js';
+import exif from './exif.js';
+import tagNames from './exif-tags.js';
+import simplify from './simplify.js';
 
 function ExifResult(startMarker, tags, imageSize, thumbnailOffset, thumbnailLength, thumbnailType, app1Offset) {
 	this.startMarker = startMarker;
@@ -103,11 +104,7 @@ Parser.prototype = {
 			thumbnailLength,
 			thumbnailType,
 			app1Offset,
-			tagNames,
 			getTagValue, setTagValue;
-		if(flags.resolveTagNames) {
-			tagNames = require('./exif-tags');
-		}
 		if(flags.resolveTagNames) {
 			tags = {};
 			getTagValue = function(t) {
@@ -199,6 +196,4 @@ Parser.prototype = {
 	}
 };
 
-
-
-module.exports = Parser;
+export default Parser;

--- a/lib/simplify.js
+++ b/lib/simplify.js
@@ -1,5 +1,5 @@
-var exif = require('./exif');
-var date = require('./date');
+import exif from './exif.js';
+import date from './date.js';
 
 var degreeTags = [{
 	section: exif.GPSIFD,
@@ -38,7 +38,7 @@ var dateTags = [{
 	name : 'ModifyDate',
 }];
 
-module.exports = {
+export default {
 	castDegreeValues: function(getTagValue, setTagValue) {
 		degreeTags.forEach(function(t) {
 			var degreeVal = getTagValue(t);

--- a/mod.ts
+++ b/mod.ts
@@ -1,9 +1,6 @@
-import { createRequire } from "https://deno.land/std@0.88.0/node/module.ts";
 import { Buffer } from "https://deno.land/std@0.88.0/node/buffer.ts";
-
-const require = createRequire(Deno.mainModule);
-const Parser = require("./lib/parser.js");
-const NodeBufferStream = require("./lib/bufferstream");
+import Parser from './lib/parser.js';
+import NodeBufferStream from './lib/bufferstream.js';
 
 export interface IExifResult {
   startMarker: { openWithOffset: Function; offset: number };

